Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.Reflection.PortableExecutable;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson4
+    {
+        public static float globalHeight = 0f;
+        public static float globalWeight = 0f;
+        public static string bmiError = "Failed calculating BMI. Reason:";
+
+        public static string Information()
+        {
+            Console.WriteLine("Testing");
+            string firstName = Console.ReadLine();
+            string errorMsg = "Name cannot be empty.";
+
+            if (String.IsNullOrEmpty(firstName))
+            {
+                Console.WriteLine(errorMsg);
+                return errorMsg;
+            }
+            else
+            {
+                return firstName;
+            }
+        }
+
+        public static int AskAge()
+        {
+            Console.WriteLine("Testing");
+            string sAge = Console.ReadLine();
+
+            int age;
+            bool isNumber = int.TryParse(sAge, out age);
+
+            if (!isNumber) return -1;
+            if (age == 0) return 0;
+            if (age >= 0) return age;
+
+            return 0;
+
+        }
+
+        public static float Weight()
+        {
+            Console.WriteLine("What is your weight in kgs?");
+            string sWeight = Console.ReadLine();
+
+            bool isNumber = float.TryParse(sWeight, out globalWeight);
+
+            if (!isNumber)
+                {
+                    Console.WriteLine(globalWeight + " is not a valid number.");    
+                    return -1; 
+                }
+            if (globalWeight == 0) return 0;
+            if (globalWeight >= 0) return globalWeight;
+
+            return 0;
+        }
+
+        public static float Height()
+        {
+            Console.WriteLine("Testing");
+            string sHeight = Console.ReadLine();
+
+            bool isNumber = float.TryParse(sHeight, out globalHeight);
+
+            if (!isNumber)
+            {
+                Console.WriteLine("\"" + sHeight + "\"" + " is not a valid number.");
+                return -1;
+            }
+            if (globalHeight == 0) return 0;
+            if (globalHeight >= 0) return globalHeight;
+
+            return 0;
+        }
+
+        public static float BMIFormula(float weight, float height)
```

Comment:
Change noun to a verb. CalculateBmi. note that I didn't write Bmi, becasue we use PascalCase even for abreviations (not BMI but Bmi)

Reply:
Thank you for the feedback, renamed this method to PromptBmi